html/partials/users.html.haml

```diff
@@ -0,0 +1,30 @@
+- if users
+  %div
+    %a.scope_anchor{:id => "users"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace